scene/gui/tree.h

```diff
@@ -527,6 +530,9 @@ friend class TreeItem;
 
 	void set_value_evaluator(ValueEvaluator *p_evaluator);
 
+	void set_delayed_text_editor_enabled(bool enabled);
+	bool get_delayed_text_editor_enabled() const;
```

Comment:
i think there is another option in tree, where you can choose if you want
doubleclick to edit, maybe we should make an enum with options like

EDITOR_ON_CLICK
EDITOR_ON_CLICK_SELECTED
EDITOR_ON_CLICK_SELECTED_DELAYED


On Fri, Jul 29, 2016 at 4:35 PM, Daniel Ramirez <notifications@github.com>
wrote:

> In scene/gui/tree.h
> <https://github.com/godotengine/godot/pull/5956#discussion_r72848784>:
>
> > @@ -527,6 +530,9 @@ friend class TreeItem;
> >
> >  	void set_value_evaluator(ValueEvaluator *p_evaluator);
> >
> > +	void set_delayed_text_editor_enabled(bool enabled);
> > +	bool get_delayed_text_editor_enabled() const;
>
> well those are shorter and basically used only in the engine so will
> change to the proposed ones
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/godotengine/godot/pull/5956/files/cf568acd52f15e2a90f3b7ec89013499cd73f3f1#r72848784>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF-Z26Z-_IdFM3amyXfIDYZvb2gKEF_cks5qalYZgaJpZM4JXx3i>
> .
>